src/js/experiments/gettingStartedGuide.js

```diff
@@ -1,5 +1,27 @@
 // https://app.optimizely.com/v2/projects/16812830475/experiments/21268663100/variations
 
+/**
+ * Changes "Your First Green Build" in the docs sidebar to "Quickstart Guide"
+ * Adds a "NEW" badge to the sidebar item as well.
+ **/
+function addNewBadgeToSidebar() {
+  const isGettingStartedPage =
+    window.location.pathname == '/docs/2.0/getting-started/' ||
+    window.location.pathname == '/docs/ja/2.0/getting-started/';
```

Comment:
this is wrong because the new badge should show up regarding of the current page?

Reply:
No, this is just being used to determine if we are on the getting started page so that we can show that the link is active.

Reply:
/docs will need to be changed to branch name in order for it to work in the preview branch but the issue is that the badge doesn't show up in the preview branch despite treatment should always be turned on during preview

Reply:
@teesloane oh! yeah I misread the code below 🤦‍♂️